amqp-0-9-1/README.md

```diff
@@ -0,0 +1,52 @@
+# java.client
+
+# About This Project
+
+The java.client project implements the AMQP protocol in Java. It provides an AMQP client API that enables developers to build Java applications that communicate with an AMQP broker over WebSocket via an RFC-6455 endpoint, such as KAAZING Gateway.
+
```

Comment:
Should "java.client" be "amqp-0-9-1" to match the other modules? Also consider specifying the AMQP version for clarity.